src/codegen/build.sbt

```diff
@@ -1,4 +1,5 @@
 //> DependsOn: core
+//> DependsOn: fuzzing
```

Comment:
OK, given that we want to move codegen into its own tool, will adding fuzzing make that harder?